flow-nodes/api-builder-plugin-logger/.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
```

Comment:
It is good practice to ensure that every file ends in a newline.

Reply:
Added the new line.